drf_spectacular/views.py

```diff
@@ -81,12 +82,23 @@ class SpectacularSwaggerView(APIView):
 
     @extend_schema(exclude=True)
     def get(self, request, *args, **kwargs):
+        swagger_bundle_js_href = "{}/swagger-ui-bundle.js".format(spectacular_settings.SWAGGER_UI_DIST)
+        swagger_bundle_css_href = "{}/swagger-ui.css".format(spectacular_settings.SWAGGER_UI_DIST)
+        swagger_bundle_favicon_href = "{}".format(spectacular_settings.FAVICON_HREF)
+
         schema_url = self.url or reverse(self.url_name, request=request)
         if request.GET.get('lang'):
             schema_url += f'{"&" if "?" in schema_url else "?"}lang={request.GET.get("lang")}'
-        settings = json.dumps(spectacular_settings.SWAGGER_UI_SETTINGS)
+        swagger_ui_settings = {'url': schema_url}
```

Comment:
not necessary, as it gets overridden automatically if present in `swagger_ui_settings`.